src/reanimated2/__mocks__/NativeReanimated.ts

```diff
@@ -6,6 +6,7 @@ const NOOP = () => {
   // noop
 };
 
+// ts-prune-ignore-next
```

Comment:
Why do we need this?

Reply:
This line is not in use, so we have to disable ts-prune for it with this directive.

Reply:
We never import `default` from this file, so it gets marked by ts-prune

Reply:
Why? It is not used, we keep it in advance, so IMHO it is fine

Reply:
We can add the whole file to list of ignored patterns as well